### PR TITLE
Improve shard modes' docs

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

File: cmd/neofs-cli/modules/control/shards_set_mode.go [hunk 1]
```diff
@@ -39,12 +44,14 @@ func initControlSetShardModeCmd() {
...
+	modes := make([]string, 0, len(mShardModes))
+	for strMode := range mShardModes {
+		modes = append(modes, "'"+strMode+"'")
+	}
+
 	flags.String(shardModeFlag, "",
-		fmt.Sprintf("New shard mode keyword ('%s', '%s', '%s')",
-			shardModeReadWrite,
-			shardModeReadOnly,
-			shardModeDegraded,
```
Comment (fyrchik) [inline cmd/neofs-cli/modules/control/shards_set_mode.go:46]: BTW what about removing `degraded` from help? It is not really intended to be used (`degraded-read-only` should) as it has it's has it's own problems. So we still support it but not suggest it to a user.

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/control/shards_set_mode.go:46]: Done

File: cmd/neofs-cli/modules/control/shards_set_mode.go [hunk 2]
```diff
@@ -36,12 +83,15 @@ func initControlSetShardModeCmd() {
 	flags := setShardModeCmd.Flags()
 	flags.StringSlice(shardIDFlag, nil, "List of shard IDs in base58 encoding")
 	flags.Bool(shardAllFlag, false, "Process all shards")
+
+	modes := make([]string, 0)
+
+	iterateSafeShardModes(func(strMode string) {
+		modes = append(modes, "'"+strMode+"'")
```
Comment (carpawell) [inline cmd/neofs-cli/modules/control/shards_set_mode.go:90]: use `fmt` pkg here?

**Review by carpawell — APPROVED** [review]

File: cmd/neofs-cli/modules/control/shards_set_mode.go (hunk 2 above)
Comment (fyrchik) [reply cmd/neofs-cli/modules/control/shards_set_mode.go:90]: It is shorter like this

Comment (carpawell) [reply cmd/neofs-cli/modules/control/shards_set_mode.go:90]: yes but looks not that obvious (more "nonhuman" symbols like `"`, `+` IMO)

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/control/shards_set_mode.go:90]: Here we just wanna wrap string into apostrophies, I don't think formatting is worth it.

File: docs/storage-node-configuration.md [hunk 3]
```diff
@@ -167,13 +167,14 @@ Contains configuration for each shard. Keys must be consecutive numbers starting
...
-| `writecache`      | [Writecache config](#writecache-subsection) |               | Write-cache configuration.               |
-| `metabase`        | [Metabase config](#metabase-subsection)     |               | Metabase configuration.                  |
-| `blobstor`        | [Blobstor config](#blobstor-subsection)     |               | Blobstor configuration.                  |
-| `gc`              | [GC config](#gc-subsection)                 |               | GC configuration.                        |
+| Parameter         | Type                                        | Default value | Description                                                                                               |
+|-------------------|---------------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------|
+| `mode`            | `string enum`                               | `read-write`  | Shard Mode.<br/>Possible values:  `read-write`, `read-only`, `degraded`, `degraded-read-only`, `disabled` |
```
Comment (fyrchik) [inline docs/storage-node-configuration.md:172]: I think `string` is enough, "possible values" already tells us that this is an enum.

File: cmd/neofs-cli/modules/control/shards_set_mode.go [hunk 4]
```diff
@@ -36,12 +83,15 @@ func initControlSetShardModeCmd() {
...
+	})
+
 	flags.String(shardModeFlag, "",
-		fmt.Sprintf("New shard mode keyword ('%s', '%s', '%s')",
-			shardModeReadWrite,
-			shardModeReadOnly,
-			shardModeDegraded,
-		),
+		fmt.Sprintf("New shard mode keyword (%s)", strings.Join(modes, ", ")),
```
Comment (fyrchik) [inline cmd/neofs-cli/modules/control/shards_set_mode.go:94]: While we are here, what about removing `keyword` from the message?

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/control/shards_set_mode.go]: Done.

File: docs/storage-node-configuration.md (hunk 3 above)
Comment (cthulhu-rider) [reply docs/storage-node-configuration.md]: Fixed.

**Review by fyrchik — APPROVED** [review]

**Pull request merged**